strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/resources/junit-platform.properties

```diff
@@ -1,3 +1 @@
-junit.jupiter.execution.parallel.enabled=true
-junit.jupiter.execution.parallel.config.strategy=fixed
-junit.jupiter.execution.parallel.config.fixed.parallelism=2
+junit.jupiter.execution.parallel.enabled=true
```

Comment:
Would you mind clarifying why you've removed these properties? Thanks! :)

Reply:
```
junit.jupiter.execution.parallel.config.strategy=fixed
junit.jupiter.execution.parallel.config.fixed.parallelism=2
```

this settings affects on behavior of `BaseStream.parallel()` from JDK, so I suggest not to use it